src/main/java/org/gatorgradle/config/GatorGradleConfig.java

```diff
@@ -89,14 +90,16 @@ private GatorGradleConfig(Path configFile) {
    * @param breakBuild     should the build fail on check failures
    * @param fastBreakBuild should the build immediately fail on check failures
    * @param assignmentName the assignment name
+   * @param usernameEnvVar should the report get user info from env var
    * @param commands       the list of commands to run
    */
   public GatorGradleConfig(boolean breakBuild, boolean fastBreakBuild, String assignmentName,
-      Collection<Command> commands) {
+      boolean usernameEnvVar, Collection<Command> commands) {
```

Comment:
Only required parameters should be in this constructor -- in fact, perhaps we should remove everything except `assignmentName` and `commands` from this one, and just use defaults in the declaration for the others

Reply:
On second thought, it appears that this constructor was actually meant to be used to specify all the values. I think it may be best to remove this constructor entirely, since it's not even used anywhere.